src/applications/appeals/996/pages/informalConferenceRep.js

```diff
@@ -2,7 +2,8 @@ import PhoneNumberWidget from 'platform/forms-system/src/js/widgets/PhoneNumberW
 import PhoneNumberReviewWidget from 'platform/forms-system/src/js/review/PhoneNumberWidget';
 import emailUI from 'platform/forms-system/src/js/definitions/email';
 
-import { errorMessages, MAX_LENGTH } from '../constants';
+import { errorMessages } from '../constants';
+import { MAX_LENGTH } from '../../shared/constants';
```

Comment:
This `MAX_LENGTH` is checking rep info, address, phone & email constants, so it's only in the 996 constants

Reply:
996 constants is what's in shared.